src/rosdistro/distribution_cache_generator.py

```diff
@@ -113,6 +113,31 @@ def generate_distribution_cache(index, dist_name, preclean=False, ignore_local=F
     return cache
 
 
+class CacheYamlDumper(yaml.SafeDumper):
+    def __init__(self, *args, **kwargs):
+        """ Allow long lines and direct unicode representation. This avoids writing escape sequences,
+            line continuations, and other noise into the cache file. """
```

Comment:
I now the existing code base is not really "clean" but it would be nice for new docblocks to follow PEP257.